draft-ietf-quic-transport.md

```diff
@@ -768,7 +780,7 @@ constant:
 
 * the location and size of the Packet Number field in long headers, and
```

Comment:
I thought I removed this from the invariants?  If I missed it, shouldn't it be removed now?